site/content/docs/manifest/worker-service.en.md.md

```diff
@@ -0,0 +1,106 @@
+List of all available properties for a `'Worker Service'` manifest. To learn about Copilot services, see the [Services](../concepts/services.en.md) concept page.
```

Comment:
Is there an extra `.md` on the file name?

Reply:
🤦‍♂️